spec/report_runner_spec.rb

```diff
@@ -24,10 +24,16 @@
       expect(report_runner.errors).to include("is missing auth_file")
     end
 
-    it "will have an error without auth_file" do
-      report_runner = ReportRunner.new
+    it "will have an error if given 'google-sheets' as an output but no drive_parent_id" do
+      report_runner = ReportRunner.new(output: 'google-sheets')
+      report_runner.valid?
+      expect(report_runner).to_not be_valid
+      
+      expect(report_runner.errors).to include("requires a drive_parent_id for google-sheets output")
+
+      report_runner.google_parent_id = "nowihaveavalue"
```

Comment:
Now you're adding a `google_parent_id` shouldn't the next line be valid?

Reply:
Okay — you explained this. The Runner is still not valid, but it is for other reasons. I think maybe a comment saying it's still invalid may have helped me, but I should have read the code more carefully.

Reply:
We discussed this in person but for posterity - we agreed...it's still invalid but now its errors array doesn't include the message about `google_parent_id`

Reply:
I was 15s ahead of you! :-D